eclipse-theia/theia
# Support plugin copying for remote feature

#### What it does

This change adds support for copying plugins from the host system to the remote system. Works in two ways:

1. Builtin plugins are transmitted directly through SSH
2. User installed plugins are installed via the new `--install-plugins` argument. This new runtime CLI argument automatically installs the specified extensions from open-vsx.

#### How to test

0. (optional) install some plugins from open-vsx.
1. Use the SSH remote feature to connect to a remote system.
2. Both builtin and manually installed plugins should have been transmitted to the remote system. You can open the vsx-registry view to confirm this.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
